### PR TITLE
Fix "make dist" wrt. doc

### PR DESCRIPTION
**Pull request by jnpkrn**
Follow-up for https://github.com/ClusterLabs/pacemaker/pull/1707#issuecomment-490132444

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -62,9 +62,10 @@ AC_ARG_WITH(pkg-name,
 
 dnl 1.11:           minimum automake version required
 dnl foreign:        don't require GNU-standard top-level files
+dnl tar-pax:        use POSIX variant of generated tar rather than v7
 dnl silent-rules:   allow "--enable-silent-rules" (no-op in 1.13+)
 dnl subdir-objects: keep .o's with their .c's (no-op in 2.0+)
-AM_INIT_AUTOMAKE([1.11 foreign silent-rules subdir-objects])
+AM_INIT_AUTOMAKE([1.11 foreign tar-pax silent-rules subdir-objects])
```
Comment (kgaillot) [inline configure.ac]: "As of 2018, this format is supported by the native tar command only on GNU, FreeBSD, OpenBSD system; it is not supported by the native tar command on NetBSD, AIX, HP-UX, Solaris." I think tar-ustar would be a better choice for the near future.

**Comment by jnpkrn** [comment]
On 10/05/19 07:42 -0700, Ken Gaillot wrote:
> "As of 2018, this format is supported by the native tar command only
> on GNU, FreeBSD, OpenBSD system; it is not supported by the native
> tar command on NetBSD, AIX, HP-UX, Solaris." I think tar-ustar would
> be a better choice for the near future.

Well, you should tell that to GitHub from where we claim the official,
blessed tarballs (or "file" utility suffers from some ambiguities,
which seems unlikely).

IOW. not worth any further micromanagement, it's quite hypocritical
when we hurt those platforms right away per what you say -- although
I had no issues with OpenIndiana, for instance, so mileage may
vary, apparently, not to speak about "make dist" not being used
for ages, apparently.

-- 
Jan (Poki)

**Comment by kgaillot** [comment]
> Well, you should tell that to GitHub from where we claim the official, blessed tarballs (or "file" utility suffers from some ambiguities, which seems unlikely). IOW. not worth any further micromanagement, it's quite hypocritical when we hurt those platforms right away per what you say -- although I had no issues with OpenIndiana, for instance, so mileage may vary, apparently, not to speak about "make dist" not being used for ages, apparently.

It's a trivial change that causes no problems and improves the situation for our users. Please make it.

We don't support portability across non-Linux systems because we don't have the capacity to do so. But we gladly accept patches from users of those systems to improve the portability, and whenever we have a trivial way to take one more obstacle out their path, it's worthwhile to do so.

**Comment by jnpkrn** [comment]
Done.

Played on the devil's advocate note, since I did actually check what
GitHub offers beforehand -- it will be the same with any git-based
(`git archive` based) solution, since git uses `pax` metadata to
store hash commit of the tree that got archived.

Also note that it seems the data format between `ustar` and `pax`
is exactly the same, except for such additional metadata (which
we have no use for, since we have only ASCII-named files, etc.,
and 99+ path length is already solved `ustar`), in which case those
metadata would get unpacked as regular files with `pax` unaware 
(`ustar` aware) implementation.

So it might be tempting to flip it back, but let's keep it at
`ustar` as you suggest.

For completeness, relevant `automake` commit:
* https://git.savannah.gnu.org/cgit/automake.git/commit/?id=fab4fb3aabf0a20d4c99fb7171de163d42e391e2

**Pull request merged**